### PR TITLE
[CSM-301] Rename params for new hierarchy

### PR DESCRIPTION
**Pull request by Pastafarianist**

### CODE REVIEW

**Comment by Martoon-00** [comment]
Thanks, this is great work!

File: src/Pos/Wallet/Web/State/Storage.hs [hunk 1]
```diff
@@ -202,51 +202,51 @@ addRemovedAccount addr@CWAddressMeta{..} = do
...
-setWalletPassLU cAddr passLU = wsWSetInfos . ix cAddr . wsiPassphraseLU .= passLU
+setWalletPassLU cWalId passLU = wsWSetInfos . ix cWalId . wsiPassphraseLU .= passLU
 
 setWalletSyncTip :: CId Wal -> HeaderHash -> Update ()
-setWalletSyncTip cAddr hh = wsWSetInfos . ix cAddr . wsiSyncTip .= hh
+setWalletSyncTip cWalId hh = wsWSetInfos . ix cWalId . wsiSyncTip .= hh
 
 addAccountHistoryTx :: AccountId -> CTxId -> CTxMeta -> Update ()
-addAccountHistoryTx accId ctxId ctxMeta =
-    wsTxHistory . ix (aiWSId accId) . at ctxId ?= ctxMeta
```
Comment (Martoon-00) [inline src/Pos/Wallet/Web/State/Storage.hs]: However I wished you to change functions like this one under this issue )
Despite it accepts `AccountId` and called `addAccountHistoryTx`, it actually gets history of whole wallet, so I propose to change argument type

**Comment by Martoon-00** [comment]
Oh lalki, it was based on a `master` branch, I noticed that too late (
Please make a PR to `0.5` now ))

**Comment by akegalj** [comment]
@Martoon-00 was this merged to 0.5 ?

**Comment by Martoon-00** [comment]
+

**Pull request merged**